lib/puppet/parser/functions/convert_to_ensurable.rb

```diff
@@ -0,0 +1,18 @@
+#
+# convert_to_ensurable.rb
+#
+# Simple converter function to map boolean values into their 'ensurable' counterparts, :present & :absent
+#
+module Puppet::Parser::Functions
```

Comment:
I would rather see this kind of function in stdlib so anyone can reuse it and not put functions like this in puppetlabs-rabbitmq.